assets/quick-order-list.js

```diff
@@ -592,9 +518,9 @@ if (!customElements.get('quick-order-list')) {
         this.updateLiveRegions(id, message);
       }
 
-      cleanErrors() {
-        this.querySelectorAll('.desktop-row-error').forEach((error) => error.classList.add('hidden'));
-        this.querySelectorAll(`.variant-item__error-text`).forEach((error) => (error.innerHTML = ''));
+      cleanErrors(id) {
+        // this.querySelectorAll('.desktop-row-error').forEach((error) => error.classList.add('hidden'));
```

Comment:
This is commented out for now given that there is clamping happening + no errors returned from the BE. More info here: https://github.com/Shopify/shopify/issues/440605